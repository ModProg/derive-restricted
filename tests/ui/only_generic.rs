#[derive_restricted::derive_where(T)]
struct Test<T>(T);

fn main() {}
