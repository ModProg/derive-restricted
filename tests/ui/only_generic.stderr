error: No supported traits found. At least one trait is required.
 --> tests/ui/only_generic.rs:1:1
  |
1 | #[derive_where::derive_where(T)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `derive_where::derive_where` (in Nightly builds, run with -Z macro-backtrace for more info)
